ui/components/ui/currency-input/currency-input.container.js

```diff
@@ -1,7 +1,7 @@
 import { connect } from 'react-redux';
 import { ETH } from '../../../helpers/constants/common';
 import { getShouldShowFiat } from '../../../selectors';
-import CurrencyInput from './currency-input.component';
+import CurrencyInput from './currency-input';
```

Comment:
Is this container used anymore? If not, the file should be deleted.

Reply:
Also the test.

Reply:
We should use selectors to get redux state in `currency-input.js` rather than keep this container and pass through connect.

Reply:
Yeah, I think this is a good comment @filipsekulic 

The refactor you've made basically takes the update of the component almost all the way to our latest patterns. Using `useSelector` within the component and deleting the `.container` file would complete the refactor

Reply:
Fixed